socorro/external/fs/crashstorage.py

```diff
@@ -17,7 +17,7 @@
     FileDumpsMapping,
     MemoryDumpsMapping,
 )
-from socorro.lib.libdatetime import utc_now, JsonDTISOEncoder
```

Comment:
We had two json encoders that had different things they did with datetimes. That's silly. We only need one and we should have everything unify on that.

This removes the second one.